modules/custom/az_paragraphs/az_paragraphs.install

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * @file
+ * Az_paragraphs_cards.install.
+ *
+ * Install, update and uninstall functions for az_paragraphs_cards install
+ * profile.
+ */
+
+use Drupal\paragraphs\Entity\Paragraph;
+
+/**
+ * Update card deck breakpoints for tablet and desktop media queries.
+ */
+function az_paragraphs_update_9201() {
```

Comment:
We actually want to add some digits for the Quickstart minor release number here per https://github.com/az-digital/az_quickstart/blob/main/RELEASES.md#update-hook-numbering-convention

01 - if we want to backport to both 2.2.x and 2.1.x
02 - if we only want to backport to 2.2.x
03 - if this should only be included in the 2.3.0 minor release

```suggestion
function az_paragraphs_update_920101() {
```